### DISCUSSION TITLE
.DS_store in repo

### DISCUSSION THREAD

**Discussion by mkleinschnitker** [blocklistproject/Lists#248] [Ideas]
Hello, 

why is a .DS_Store part of the repository? I would suggest to add .DS_Store files to .gitignore file.

**Comment by gap579137** [comment]
TL;DR - Don't know what that is but will google it LOL

As the lead maintainer of the project I have to admit I am still learning about GitHub. When we first started this project just for our group we were just hosting it on a vps in the cloud. When it got to where we were blowing through our monthly bandwidth budget from our provider we had to figure something out. The community as a whole suggested that we move the project over to GitHub. I initially cringed because I would have to admit I didn't know what I was doing on GH. As the project has matured we have been able to figure out what we are doing, typically with the help of Google and Youtube. All that to say I honestly don't know what a .gitignore file is but can infer that it is a file that tells GH to ignore certain files. I'll look it up and figure out how to do it.

**Comment by gap579137** [comment]
Found this great site that help explain it -> [https://www.pluralsight.com/guides/how-to-use-gitignore-file](https://www.pluralsight.com/guides/how-to-use-gitignore-file)
